### ISSUE TITLE
Figure out convergence conditions for sparse.linalg.eigsh

### ISSUE THREAD

**Issue by jni** [elegant-scipy/elegant-scipy#130]
This error happens sporadically in ch6:

``` python
Affn = Dinv2 @ lap @ Dinv2
eigvals, vec = sparse.linalg.eigsh(Affn, k=3, which='SM')
_, x, y = (Dinv2 @ vec).T
------------------

ArpackNoConvergence: ARPACK error -1: No convergence (30001 iterations, 0/3 eigenvectors converged)
```

@stefanv do you know what causes this and how to avoid it? Would more iterations help, or a more intelligent choice for the starting vector?

**Comment by stefanv** [comment]
@jni Is this solved?

**Comment by jni** [comment]
@stefanv yes and no. Yes because that whole section ended in the scrap heap. No because we ended up with a couple of hacks in there that sort of made it work sometimes on some of our machines, but we never really understood it enough to have it properly fixed. I'll leave this here for 2nd edition but remove the urgent tags. =)
